thorlcr/activities/lookupjoin/thlookupjoin.cpp

```diff
@@ -69,6 +72,19 @@ class CLookupJoinActivityMaster : public CMasterActivity
             serializeMPtag(dst, rhsDistributeTag);
         }
     }
+    virtual void deserializeStats(unsigned node, MemoryBuffer &mb) override
+    {
+        CMasterActivity::deserializeStats(node, mb);
+
+        if (isSmartJoin(*this))
+            mb.read(reinterpret_cast<std::underlying_type<SmartJoinState>::type &> (smartState));
```

Comment:
This isn't really right.  If it is local you will have one value per node - but this is saving one value globally.  More useful for local would be the NumStandardJoin, v NumLocalJoin.
Also, what happens if this is used in a LOOP - it only records the value from the last iteration whilst Counts would provide a bit more information.